tests/functional/pages/surveys/question.page.js

```diff
@@ -33,6 +33,11 @@ class QuestionPage {
     return browser.click('a[id="top-previous"]')
   }
 
+  next() {
```

Comment:
Submit verifies that you're on the correct page. The next allows me to bypass this check. The alternative would be to import 40 page objects - which is overkill for this test. All I care about is navigating through to the thank you page so this is the simplest way of doing that.